test/exfmt/integration/map_test.exs

```diff
@@ -47,11 +47,11 @@ defmodule Exfmt.Integration.MapTest do
   end
 
   test "chained map get" do
-    assert_format "map.key.another.a_third\n"
+    "map.key.another.a_third" ~> "map.key().another().a_third()\n"
```

Comment:
I'm unsure about this. I would lean towards not adding parens here as we don't know if it's getting a value from a map or calling a function.

I've left a comment here for guidance: https://github.com/lexmag/elixir-style-guide/issues/40#issuecomment-322306851